rules/windows/process_creation/proc_creation_win_powershell_decode_gzip.yml

```diff
@@ -0,0 +1,20 @@
+title: Gzip Archive Decode Via PowerShell
+id: 98767d61-b2e8-4d71-b661-e36783ee24c1
+status: experimental
+description: Detects attempts of decoding encoded Gzip archive via PowerShell.
+references:
+    - https://www.zscaler.com/blogs/security-research/onenote-growing-threat-malware-distribution
+author: Hieu Tran
+date: 2023/03/13
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection:
+        CommandLine|contains|all:
+            - 'GZipStream'
+            - '::Decompress'
+    condition: selection
+falsepositives:
+    - Legitimate administrative scripts may use this functionality. Use "ParentImage" in combination with Script names and allowed users and applications to filter legitimate executions
+level: medium
```

Comment:
Discussion: I am not sure about the medium level. This rule is more of a "feature used" tagging than actual indicator of maliciousness. Any thoughts?

Reply:
cc: @nasbench @frack113

Reply:
It's a "feature" that can be suspicious or benign hence it's medium. Low rules are more for enrichment and are very noisy (which doesn't apply necessarily here).

Once the hunting rule is public we will transfer these kinds of rules there.